app/_locales/en/messages.json

```diff
@@ -714,8 +714,7 @@
     "message": "This Chain ID is currently used by the $1 network."
   },
   "chainListReturnedDifferentTickerSymbol": {
-    "message": "This network doesn’t match its associated chain ID or name. Many popular tokens use the name $1, making it a target for scams. Scammers may trick you into sending them more valuable currency in return. Verify everything before you continue.",
```

Comment:
we also need to remove `chainListReturnedDifferentTickerSymbol` in other languages so it would generate a new translated string for us.

Reply:
fixed! thanks @DDDDDanica